tests/test_utils.py

```diff
@@ -59,6 +59,11 @@ def test_make_browse_image():
     utils.make_browse_image(input_tif, output_png)
     assert open(output_png, 'rb').read() == open('tests/data/test_browse_image.png', 'rb').read()
     os.remove(output_png)
+    input_empty = 'tests/data/test_empty.tif'
```

Comment:
```suggestion

    input_empty = 'tests/data/test_empty.tif'
```
Nice to put a little white space between the different test blocks, or, even put it into it's own test like:
```
def test_make_browse_image_geotiff_all_nodata():
```

Reply:
No problem, I made a new test for it